src/amo/components/Home/styles.scss

```diff
@@ -58,6 +58,20 @@
   }
 }
 
+.listTitle {
```

Comment:
We tend to prefix the CSS class names by the component name. In your case, the CSS class is inside the `Home-SubjectShelf-link` block, so I'd reuse it. For example: `.Home-SubjectShelf-link-text`.